app/components/DayPickerInput/styles.css

```
@@ -20,3 +20,8 @@
   .DayPicker-Day:not(.DayPicker-Day--disabled):not(.DayPicker-Day--selected):not(.DayPicker-Day--outside):hover {
   background-color: #0f172a;
 }
+
```

To match color of selected day.

```suggestion
.DayPicker-TodayButton {
  color: #3b82f6;
}
```